stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -30,21 +30,25 @@ const SecuritiesTable = () => {
 
   return (
     <div className="securities-container">
-      <div className="securities-title">My Securities Table</div>
+      <div className="securities-title-container">
+        <div className="securities-title">My Securities Table</div>
+        <div className="add-securities-button-above-table">
+          {securitiesData.length > 0 && <AddSecurityButton />}
+        </div>
+      </div>
       <div className="header-container">
         <div className="securities-table-header">Exchange</div>
         <div className="securities-table-header">Symbol</div>
         <div className="securities-table-header">Name</div>
+        <div className="securities-table-header">Edit / Delete</div>
       </div>
       {isLoading ? (
         <div className="spinner-container">
           <Components.LargeSpinner />
         </div>
       ) : errorMessage ? (
-        <div className="securities-error-message">
-          {errorMessage}
-        </div>
-        ) : (
+        <div className="securities-error-message">{errorMessage}</div>
```

Comment:
This could be extracted into a separate component very easily. Just pass in the `errorMessage` as a prop.

Reply:
Okay done this now